scenes/gods/god.gd

```diff
@@ -0,0 +1,6 @@
+extends Node2D
+
+@export var god_resource: RGod
+
+func _ready() -> void:
```

Comment:
We should probably do this assert in the init, and then also pass in the god resource as paramter to we can God.new(god_resource) when we instantiate them.

Not blocking but should try to change in future submits.